paper.md

```diff
@@ -95,6 +95,10 @@ Nonetheless, these resources are often not available open-source and can thus no
 Also, these resources are intended to use for self-study. Our material can be used for self-study, but it is primarily developed for instructors to use in a workshop.
 In addition, although a diverse range of online courses already exists, few are targeted towards academic researchers.
 
+There is another Carpentries lesson on deep learning: Introduction to artificial neural networks in Python (@carpentries-ann-python).
```

Comment:
In case you change the BibTex above, this reference also needs to be adapted:
```suggestion
There is another Carpentries lesson on deep learning: Introduction to artificial neural networks in Python (@Pollard_Introduction_to_artificial_2022).
```